eng/targets/Helix.props

```diff
@@ -35,4 +36,14 @@
     <HelixPreCommand Include="call RunPowershell.cmd mssql\InstallSqlServerLocalDB.ps1 || exit /b 1" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TestDependsOnNode)' == 'true' AND '$(IsWindowsHelixQueue)' == 'false'">
+    <HelixPreCommand Include="./installnode.sh $(NodeVersion) linux-x64" />
+    <HelixPreCommand Include="export PATH=%24PATH&quot;:&quot;%24HELIX_CORRELATION_PAYLOAD/node/bin" />
```

Comment:
Question, mostly out of curiosity. Could we symlink the node executable into some kind of HELIX_BIN directory? This is the 'normal' way to install things in *nix.

Reply:
That sounds interesting, so are you thinking we create some kind of `/bin` that we put any things that we want to be in the path for runtests, and add that well known `/bin` to the path?  Then we could just add a symlink to that directory instead what i'm currently doing.  Well I guess it'd be the same just using a well known bin instead of a special node dir.